client/boot/app.js

```diff
@@ -0,0 +1,60 @@
+// Initialize localStorage polyfill before any dependencies are loaded
+require( 'lib/local-storage' )();
+if ( process.env.NODE_ENV === 'development' ) {
+	require( 'lib/wrap-es6-functions' )();
```

Comment:
Yes, it won't be part of production build. I like this feature of Webpack a lot 💯